src/blocks/stars.css

```diff
@@ -0,0 +1,31 @@
+/*#region Rate and reviews*/
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin: 0 4px 0 0;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url("../images/star.svg");
+}
+
+.stars--1 :nth-child(-n+1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
+  background-image: url("../images/star-active.svg");
+}
+
+.stars__reviews {
+  margin-left: 17px;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-color-bold);
+}

```

Comment:
Here I suppose you can't modify BEM, block. I think you need to add wrap <div > and use BEM mix

Reply:
it is not about external geometry, so bem mixes are useless